examples/mobile/UIComponents/components/view-controls/main-tab/main-tab-4-items.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page">
+		<div class="ui-content ui-tabs">
+			<div class="ui-section-changer">
+				<div>
+					<section class="ui-section-active">
+						<div class="ui-content-area" style="height: 100%; padding: 22px;">
```

Comment:
Isn't it more clear to add an additional class with height and padding instead of inline styles?

Reply:
I also have a question.
What's mean the  "padding:22px"
I can't find the value in the Guide.

And if it's possible, it's better put the value into the tau.css not sample code.

Reply:
I understand your confusion. This style is not defined in guideline so I cannot move it to the framework. I added this title and style for Tab because user doesn't see what happens when he change Tab. I myself had doubts whether to do it

Reply:
I added "app" css class instead of inline styles. I removed padding but I left "titles" for tabs for check running of bottom buttons.